generators/generator-base.js

```diff
@@ -1683,6 +1684,24 @@ module.exports = class extends PrivateBase {
     }
 
     /**
+     * Print an info message.
+     *
+     * @param {string} value - message to print
+     */
+    info(msg) {
```

Comment:
the logging related methods are kind of spread around. There is debug in base-private. may be we should bring them all in this file also may be make it more clearer by making a wrapper object so that we can do
```
this.log.info()
this.log.debug()
this.log.error()
```
and so on. WDYT?